elements/pfe-icon/src/pfe-icon.scss

```diff
@@ -31,15 +38,16 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
 }
 
 :host {
-  --context: #{pfe-local(context)};  
+  --context: #{pfe-local(context)};
 
   position: relative;
   display:  inline-block;
 
   // Supports sizing while preserving aspect ratio (despite layout)
```

Comment:
Added the additional bit about the content-box model since that is an important part of making this work and we want to make sure no one accidentally removes it in the future.